### DISCUSSION TITLE
Future graphical display update

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#140] [Ideas]
Came across this today on youtube https://www.youtube.com/watch?v=FuD-QmKn1uo

I think it would fit quite nicely with the Els Nano.  Quite pricey but hey all the best toys are :)

**Comment by AndreyFly007** [comment]
This display is too small.
![20231119_081022](https://github.com/kachurovskiy/nanoels/assets/111601202/a796167f-dde2-4f55-b8f3-c1efa63fbdc1)

**Comment by kachurovskiy** [comment]
That's a nice display but yes, a little on the small side. Preference for larger displays is something I hear often and no one has ever asked for a smaller display.

I also fear that using those fancy displays is going to balloon the code size, complicate the compilation and take 1 core from us.

**Comment by Festivejelly** [comment]
Okie dokie fair point. I think then back to the idea of having a web based interface makes sense. Ive started on the front end of it and ive created backend API in node,js to talk to the controller since not all browsers support serial to com port.

Right now im essentially emulating the button press commands and all the logic to build up the commands is done in the node API and then the controller essentially queues them up.

Im also streaming the print to LCD output back over serial to the api. Hope to release something soon.
